cluster/sbin/check-dir-option.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+#
+# Copyright (c) 2018 SnappyData, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you
+# may not use this file except in compliance with the License. You
+# may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing
+# permissions and limitations under the License. See accompanying
+# LICENSE file.
+#
+
+# Check whether "-dir" argument has been provide or not while invoing snappy-locator.sh/snappy-server.sh/snappy-lead.sh
+#
+
+noOfInputsArgs=$#
+
+if [ $noOfInputsArgs -eq 0 ]
+  then  #if no arguments passed
+	echo "ERROR : No arguments has been provided. Please provide required arguments"
```

Comment:
1) No space before :
2) 'have been' instead of 'has been'?

Reply:
Modified as per review comments